### PR TITLE
Show practice quiz/survey message in submission list

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-12972
affects: Teacher
release note: Made it more clear that you can not get to speed grader for practice quizzes and surveys

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005223&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Made it more clear that you can not get to speed grader for practice quizzes and surveys

#### Affected Apps: Teacher

[MBL-12972](https://instructure.atlassian.net/browse/MBL-12972)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 95.07% | 95.06% | 0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/src/modules/quizzes/submissions/QuizSubmissionList.js [hunk 1]
```diff
@@ -113,16 +139,15 @@ export class QuizSubmissionList extends Component<QuizSubmissionListProps, any>
   }
 
   renderRow = ({ item, index }: { item: SubmissionRowDataProps, index: number }) => {
-    let disclosure = true
-    if (this.props.quiz) {
-      disclosure = !!this.props.quiz.data.assignment_id
-    }
+    let onPress = this.props.quiz?.data.assignment_id == null
```
Comment (armstrongnate) [inline rn/Teacher/src/modules/quizzes/submissions/QuizSubmissionList.js]: Hooray for null coalescing!

**Pull request merged**